src/NodeServices/Analytics.cs

```diff
@@ -183,6 +183,7 @@ public static IDisposable TrackFileOperationEvent(string filepath, Actions opera
         /// </summary>
         /// <param name="tag">Usage tag</param>
         /// <param name="data">Usage data</param>
+        [Obsolete("Function will be deprecated in Dynamo 3.0 as Dynamo will no longer support GA instrumentation.")]
```

Comment:
I find this message confusing. Using `Obsolete` is already deprecating this function. Did you mean to say `removed` instead?
Also saying `GA instrumentation` makes it unclear if it refers to `Instrumentation` or `Google Analytics`. If I understood correctly, we are talking about `Instrumentation`, right?

Reply:
We are talking about Google instrumentation specifically. Not instrumentation in general since we most likely will soon add some instrumentation ability back.

Reply:
I changed `deprecated ` to removed